invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/components/DocumentMetadata/components/index.js

```diff
@@ -45,3 +45,13 @@ export const DocumentTitle = connect(
   mapStateToProps,
   null
 )(DocumentTitleComponent);
+
+export const DocumentMetadataTabs = connect(
+  mapStateToProps,
+  null
+)(DocumentMetadataTabsComponent);
+
+export const DocumentMetadataAccordion = connect(
```

Comment:
This is also connected here `invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Documents/DocumentsDetails/components/DocumentMetadata/index.js`